src/constants/index.ts

```diff
@@ -831,6 +915,19 @@ export const POOLS_MAP: PoolsMap = {
     route: "arbusd",
     rewardPids: buildPids({ [ChainId.ARBITRUM]: 1 }),
   },
+  [SUSD_METAPOOL_V2_NAME]: {
+    name: SUSD_METAPOOL_V2_NAME,
+    lpToken: SUSD_SWAP_V2_TOKEN,
+    poolTokens: SUSD_POOL_TOKENS,
+    addresses: SUSD_META_SWAP_V2_DEPOSIT_ADDRESSES,
+    isSynthetic: true,
+    type: PoolTypes.USD,
+    metaSwapAddresses: SUSD_META_SWAP_V2_ADDRESSES,
+    underlyingPoolTokens: SUSD_UNDERLYING_POOL_TOKENS,
+    underlyingPool: STABLECOIN_POOL_V2_NAME,
+    route: "susdv2",
+    rewardPids: buildPids({}),
```

Comment:
Were the correct properties updated here for the new V2 address values? @hammeiam

Reply:
afik there are no staking rewards for the v2 metapools. Is that correct @penandlim ?

Reply:
how does the rest of the object look as far as what has to be changed for V2?